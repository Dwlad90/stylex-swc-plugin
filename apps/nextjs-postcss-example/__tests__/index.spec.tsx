/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';
import Home from '@/app/page';

describe('Home', () => {
  it('render styles successfully', () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
