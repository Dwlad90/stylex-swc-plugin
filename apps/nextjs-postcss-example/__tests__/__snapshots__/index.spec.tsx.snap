// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render styles successfully 1`] = `
<div>
  <main
    class="x78zum5 xdt5ytf x6s0dn4 x1qughib xg6iff7 x1bomb4u xaxnm0k x3tt2s9 xmz0i5r xs6rbim xysyzu8 x7f1qth"
  >
    <div
      class="x1jfb8zj xarpa2k x1h91t0o xdnka2l x1xg1i7y xh8yej3 xhtitgo x1y8sf5l"
    >
      <p
        class="xjg0vao x1n2onr6 x15f3dyk xo5s888 xu8adaz x1v68ji2 x1ghz6dp xx7ihp2 x1876b5s x13b8fpu x14s53in x3tt2s9 x1orz5ps x130g4yn xmkeg23 x1ehvz7d x1y0btm7 xyiqvn9 xek1nhq xcsupl4 xd22jv x1los6se"
      >
        Get started by editing 
        <code
          class="x1xlr1w8 x1y8sf5l"
        >
          app/page.tsx
        </code>
      </p>
    </div>
    <div
      class="x1iyjqo2 x78zum5 xdt5ytf x6s0dn4 xl56j7k x1r7vprt x1a02dak"
    >
      <h1
        class="xs4a3dk xo5v014 x1pmkfg8 xo1l8bm x2b8uid x78zum5 xmjkwu9 xuxw1ft xdt5ytf xwlf911"
      >
        Next.js App Dir
        <span
          class="x1n2onr6 x6icuqf x13vifvy x1n8jcod x5mrdy x1c74tu6 xa4qsjk x1esw782"
        >
          ♥️
        </span>
        ️StyleX
      </h1>
      <div
        class="x78zum5 x6s0dn4 xl56j7k x1q0g3np x1azadol xmkeg23 x1y0btm7 xd1nvyh x1arqxy9 x1pmkfg8 x1ipj079"
      >
        <button
          class="x78zum5 x6s0dn4 xl56j7k x17frcva x1plog1 x1tcxl6e xiy85n xktz6yv x1umsj2k xbznm0z xc342km xng3xce xcsupl4 x1hlvdh8 x11sf7u x1ypdohk xmc08wn x1u4xmye xglsxx3"
        >
          -
        </button>
        <div
          class="xmc08wn x3stwaq xxhslpl x1843ork x2b8uid x1y8sf5l"
        >
          0
        </div>
        <button
          class="x78zum5 x6s0dn4 xl56j7k x17frcva x1plog1 x1tcxl6e xiy85n xktz6yv x1umsj2k xbznm0z xc342km xng3xce xcsupl4 x1hlvdh8 x11sf7u x1ypdohk xmc08wn x1u4xmye xglsxx3"
        >
          +
        </button>
      </div>
    </div>
    <div
      class="xq5fvzb x18mbai7 xl56j7k x78zum5 x6s0dn4 x1jwldgz xvnzane"
    >
      <div
        class="x19na6o3"
      >
        Pink rounded rectangle with blue text
      </div>
    </div>
    <div
      class="xrvj5dj xtp8ymz xx3cr9d xtffbmy x7gess9 x193iq5w xl858mc x15hltav"
    >
      <a
        class="x78zum5 x1ymzxv2 x6s0dn4 x1nhvcw1 xdt5ytf xcsupl4 x1d0mer6 xhxp53m xmkeg23 x1y0btm7 x1e3d2hy xhczobc x1heor9g x1pmkfg8 xzrlz9m xts7igz xofcydl x2b8uid x1hl2dhg x1jrvdfj xwht23z"
        href="https://stylexjs.com/docs/learn/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <h2
          class="xko2ofl x1pe7niw x1s688f x1ozsttx x4dkmba"
        >
          Docs
           
          <span
            class="x1rg5ohu x11xpdln xan5c0p x13dflua x12w9bfk"
          >
            →
          </span>
        </h2>
        <p
          class="x1ghz6dp x197sbye xtyu05a x1w2vvpw x1evy7pa x1u8bs6r"
        >
          Learn how to use StyleX to build UIs
        </p>
      </a>
      <a
        class="x78zum5 x1ymzxv2 x6s0dn4 x1nhvcw1 xdt5ytf xcsupl4 x1d0mer6 xhxp53m xmkeg23 x1y0btm7 x1e3d2hy xhczobc x1heor9g x1pmkfg8 xzrlz9m xts7igz xofcydl x2b8uid x1hl2dhg x1jrvdfj xwht23z"
        href="https://stylexjs.com/docs/api/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <h2
          class="xko2ofl x1pe7niw x1s688f x1ozsttx x4dkmba"
        >
          API
           
          <span
            class="x1rg5ohu x11xpdln xan5c0p x13dflua x12w9bfk"
          >
            →
          </span>
        </h2>
        <p
          class="x1ghz6dp x197sbye xtyu05a x1w2vvpw x1evy7pa x1u8bs6r"
        >
          Browse through the StyleX API reference
        </p>
      </a>
      <a
        class="x78zum5 x1ymzxv2 x6s0dn4 x1nhvcw1 xdt5ytf xcsupl4 x1d0mer6 xhxp53m xmkeg23 x1y0btm7 x1e3d2hy xhczobc x1heor9g x1pmkfg8 xzrlz9m xts7igz xofcydl x2b8uid x1hl2dhg x1jrvdfj xwht23z"
        href="https://stylexjs.com/playground/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <h2
          class="xko2ofl x1pe7niw x1s688f x1ozsttx x4dkmba"
        >
          Playground
           
          <span
            class="x1rg5ohu x11xpdln xan5c0p x13dflua x12w9bfk"
          >
            →
          </span>
        </h2>
        <p
          class="x1ghz6dp x197sbye xtyu05a x1w2vvpw x1evy7pa x1u8bs6r"
        >
          Play with StyleX and look at the compile outputs
        </p>
      </a>
      <a
        class="x78zum5 x1ymzxv2 x6s0dn4 x1nhvcw1 xdt5ytf xcsupl4 x1d0mer6 xhxp53m xmkeg23 x1y0btm7 x1e3d2hy xhczobc x1heor9g x1pmkfg8 xzrlz9m xts7igz xofcydl x2b8uid x1hl2dhg x1jrvdfj xwht23z"
        href="https://github.com/nmn/nextjs-app-dir-stylex"
        rel="noopener noreferrer"
        target="_blank"
      >
        <h2
          class="xko2ofl x1pe7niw x1s688f x1ozsttx x4dkmba"
        >
          Templates
           
          <span
            class="x1rg5ohu x11xpdln xan5c0p x13dflua x12w9bfk"
          >
            →
          </span>
        </h2>
        <p
          class="x1ghz6dp x197sbye xtyu05a x1w2vvpw x1evy7pa x1u8bs6r"
        >
          Get started with a NextJS+StyleX project
        </p>
      </a>
    </div>
  </main>
</div>
`;
